account_delete_transaction_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 )
 
-func TestAccountDeleteTransaction_Execute(t *testing.T) {
+func TestIntegrationAccountDeleteTransaction(t *testing.T) {
```

Comment:
incorrect test name; should have a test name, at the moment it simply is stating it's an integration tests that tests `AccountDeleteTransaction`. Something like `CanSuccessfullyExecute` at the end should be added.